bbpress/content-single-forum.php

```diff
@@ -11,7 +11,7 @@
 
 <div id="bbpress-forums">
 
-	<?php bbp_breadcrumb(); ?>
+<!--	--><?php //bbp_breadcrumb(); ?>
```

Comment:
Fair enough; this works. I guess it might be a little clearer if you put something in the comment like `<!-- RCG Changed -->`